FindUMFPACK.cmake does not set UMFPACK_* variables

From personal correspondence with Fabian Bartel:

```
CMake Error at CMake/Basis/CommonTools.cmake:840 (message):
 
 
   Module Numerics was requested to be build with UMFPACK.  Please ensure that
   the package is installed in a standard system location or set
   DEPENDS_UMFPACK_DIR to the installation prefix (i.e., root directory of the
   installation).  To disable features which require this optional dependency,
   set the WITH_UMFPACK option to OFF and try again.
 
   The DEPENDS_UMFPACK_DIR variable can alternatively be set to the directory
   containing a UMFPACKConfig.cmake or umfpack-config.cmake file.  If no such
   file exists, contact either the developer of this project or CMake BASIS to
   provide a FindUMFPACK.cmake file.
 
Call Stack (most recent call first):
   CMake/Basis/ProjectTools.cmake:2143 (basis_find_package)
   CMake/Basis/ProjectTools.cmake:2430 (basis_find_packages)
   CMake/Modules/mirtkProjectBegin.cmake:90 (basis_project_begin)
   CMake/Modules/mirtkConfigureModule.cmake:41 (mirtk_project_begin)
   Modules/Numerics/CMakeLists.txt:28 (mirtk_configure_module)
```

It seems the SuiteSparse is found correctly:
```
grep -i suitesparse CMakeCache.txt
SuiteSparse_AMD_INCLUDE_DIR:PATH=/usr/include/suitesparse
SuiteSparse_AMD_LIBRARY:FILEPATH=/usr/lib64/libamd.so
SuiteSparse_CHOLMOD_INCLUDE_DIR:PATH=/usr/include/suitesparse
SuiteSparse_CHOLMOD_LIBRARY:FILEPATH=/usr/lib64/libcholmod.so
SuiteSparse_COLAMD_INCLUDE_DIR:PATH=/usr/include/suitesparse
SuiteSparse_COLAMD_LIBRARY:FILEPATH=/usr/lib64/libcolamd.so
SuiteSparse_METIS_LIBRARY:FILEPATH=SuiteSparse_METIS_LIBRARY-NOTFOUND
SuiteSparse_RT_LIBRARY:FILEPATH=/usr/lib64/librt.so
SuiteSparse_UMFPACK_INCLUDE_DIR:PATH=/usr/include/suitesparse
SuiteSparse_UMFPACK_LIBRARY:FILEPATH=/usr/lib64/libumfpack.so
SuiteSparse_config_INCLUDE_DIR:PATH=/usr/include/suitesparse
SuiteSparse_config_LIBRARY:FILEPATH=/usr/lib64/libsuitesparseconfig.so
//ADVANCED property for variable: SuiteSparse_AMD_INCLUDE_DIR
SuiteSparse_AMD_INCLUDE_DIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: SuiteSparse_AMD_LIBRARY
SuiteSparse_AMD_LIBRARY-ADVANCED:INTERNAL=1
//ADVANCED property for variable: SuiteSparse_CHOLMOD_INCLUDE_DIR
SuiteSparse_CHOLMOD_INCLUDE_DIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: SuiteSparse_CHOLMOD_LIBRARY
SuiteSparse_CHOLMOD_LIBRARY-ADVANCED:INTERNAL=1
//ADVANCED property for variable: SuiteSparse_COLAMD_INCLUDE_DIR
SuiteSparse_COLAMD_INCLUDE_DIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: SuiteSparse_COLAMD_LIBRARY
SuiteSparse_COLAMD_LIBRARY-ADVANCED:INTERNAL=1
//ADVANCED property for variable: SuiteSparse_METIS_LIBRARY
SuiteSparse_METIS_LIBRARY-ADVANCED:INTERNAL=1
//ADVANCED property for variable: SuiteSparse_RT_LIBRARY
SuiteSparse_RT_LIBRARY-ADVANCED:INTERNAL=1
//ADVANCED property for variable: SuiteSparse_UMFPACK_INCLUDE_DIR
SuiteSparse_UMFPACK_INCLUDE_DIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: SuiteSparse_UMFPACK_LIBRARY
SuiteSparse_UMFPACK_LIBRARY-ADVANCED:INTERNAL=1
//ADVANCED property for variable: SuiteSparse_config_INCLUDE_DIR
SuiteSparse_config_INCLUDE_DIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: SuiteSparse_config_LIBRARY
SuiteSparse_config_LIBRARY-ADVANCED:INTERNAL=1
```
 
But then it does not convert this to UMFPACK variables:
```
grep -i umf CMakeCache.txt
//Top-level installation directory of UMFPACK or directory containing
// UMFPACKConfig.cmake file.
DEPENDS_UMFPACK_DIR:PATH=NOTFOUND
SuiteSparse_UMFPACK_INCLUDE_DIR:PATH=/usr/include/suitesparse
SuiteSparse_UMFPACK_LIBRARY:FILEPATH=/usr/lib64/libumfpack.so
//Build with optional UMFPACK dependency
WITH_UMFPACK:BOOL=ON
//ADVANCED property for variable: SuiteSparse_UMFPACK_INCLUDE_DIR
SuiteSparse_UMFPACK_INCLUDE_DIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: SuiteSparse_UMFPACK_LIBRARY
SuiteSparse_UMFPACK_LIBRARY-ADVANCED:INTERNAL=1
//Directory containing UMFPACKConfig.cmake file.
UMFPACK_DIR:INTERNAL=NOTFOUND
//MODIFIED property for variable: WITH_UMFPACK
WITH_UMFPACK-MODIFIED:INTERNAL=ON
//(Previous) DEPENDS_UMFPACK_DIR value.
_DEPENDS_UMFPACK_DIR:INTERNAL=NOTFOUND
//(Previous) UMFPACK_DIR value.
_UMFPACK_DIR:INTERNAL=NOTFOUND
```

--- a/CMake/Modules/FindUMFPACK.cmake
+++ b/CMake/Modules/FindUMFPACK.cmake
@@ -35,6 +35,11 @@ set(UMFPACK_VERSION_MINOR  ${SuiteSparse_VERSION_MINOR})
 set(UMFPACK_VERSION_PATCH  ${SuiteSparse_VERSION_PATCH})
 set(UMFPACK_VERSION_STRING ${SuiteSparse_VERSION_STRING})
 
+set(UMFPACK_INCLUDE_DIR  "${SuiteSparse_UMFPACK_INCLUDE_DIR}")
+set(UMFPACK_INCLUDE_DIRS "${SuiteSparse_UMFPACK_INCLUDE_DIRS}")
+set(UMFPACK_LIBRARY      "${SuiteSparse_UMFPACK_LIBRARY}")
+set(UMFPACK_LIBRARIES    "${SuiteSparse_UMFPACK_LIBRARIES}")
+
 # ------------------------------------------------------------------------------
 # Handle QUIET, REQUIRED, and [EXACT] VERSION arguments and set SuiteSparse_FOUND
 include(FindPackageHandleStandardArgs)

